### PR TITLE
Update manual testing in dg for find

### LINKED ISSUE #255: Update DG for find command AND search

### PR DESCRIPTION
**Pull request by Feng1231**
image
closes #189 
closes #255

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -480,6 +480,55 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
+### Finding students
+
+1. Finding students by name
```
[the remark this answers is not in the document]
Comment (TY1Fan) [reply docs/DeveloperGuide.md:485]: I agree. I feel that the numbering should not be all 1.

[the remark this answers is not in the document]
Comment (Feng1231) [reply docs/DeveloperGuide.md:485]: huh, didn't dan show us the convention suggested by the course website on friday?
image

[the remark this answers is not in the document]
Comment (Soumyadip-cmd) [reply docs/DeveloperGuide.md:485]: Wait where is this from?

[the remark this answers is not in the document]
Comment (teamotitan) [reply docs/DeveloperGuide.md:485]: Hmmm you are right, I apologise I missed out on the conventions

**Review by teamotitan — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (TY1Fan) [reply docs/DeveloperGuide.md:485]: > huh, didn't dan show us the convention suggested by the course website on friday? image

Oops my bad, I apologise too for missing out on this.

**Review by TY1Fan — APPROVED** [review]
LGTM

**Pull request merged**